<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan-Datei-Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl CCD Pixel in x-Richtung</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbCCDX">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl CCD Pixel in y-Richtung</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbCCDY">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbScanX">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>106</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl Pixel im Scanningbild (x-Richtung)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbScanY">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>106</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl Pixel im Scanningbild (y-Richtung)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdGenImage">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan-Datei generieren</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdOpenImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Bild Ã¶ffnen</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagePreview">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
